### ISSUE TITLE
Max line length should ignore URL in a comment

### ISSUE THREAD

**Issue by chengyin** [rrrene/credo#557]
### Environment

* Credo version (`mix credo -v`): 0.9.2
* Erlang/Elixir version (`elixir -v`): 1.6.5
* Operating system: macOS 10.13.5 (17F77)

### What were you trying to do?

Add a comment like this:

```elixir
def fun do
  # Based on https://github.com/rrrene/credo/blob/7dec9aecdd21ef33fdc20cc4ac6c94efb4bcddc3/lib/credo.ex#L4
  nil
end
```

### Expected outcome

No lint error since there is no way to add a line break in the middle of the URL.

### Actual outcome

Lint error: Line is too long.

There doesn't seem to be a config related to this.

**Comment by rrrene** [comment]
Thx for reporting! :+1:

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍
